dbms/src/Common/TiFlashMetrics.h

```diff
@@ -893,6 +900,7 @@ struct MetricFamily
     prometheus::Family<T> * store_family;
     std::vector<MetricArgType> store_args;
     // <resource_group_name, metrics>
+    std::shared_mutex mu;
     std::unordered_map<String, std::vector<T *>> metrics_map;
```

Comment:
Maybe change it to a better name?
Maybe `resource_group_metrics_map` and `resource_group_metrics_mu`

Reply:
done